config/database.yml

```diff
@@ -1,25 +1,16 @@
-# SQLite version 3.x
-#   gem install sqlite3
-#
-#   Ensure the SQLite 3 gem is defined in your Gemfile
-#   gem 'sqlite3'
-#
 default: &default
-  adapter: sqlite3
-  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  adapter: postgresql
+  encoding: unicode
+  username: <%= Rails.application.credentials.database[:postgres_user] %>
+  password: <%= Rails.application.credentials.database[:postgres_password] %>
+  pool: 5
   timeout: 5000
+  host: <%= Rails.application.credentials.database[:postgres_host] %>
```

Comment:
Is there a smaller way to get variables that is not 
`Rails.application.credentials.xxxxx`? sounds too long :(

Reply:
Marked as resolved since the logic has changed !